insights/specs/insights_archive.py

```diff
@@ -58,6 +58,7 @@ class InsightsArchiveSpecs(Specs):
     fdisk_l = simple_file("insights_commands/fdisk_-l")
     foreman_rake_db_migrate_status = simple_file('insights_commands/foreman-rake_db_migrate_status')
     getcert_list = simple_file("insights_commands/getcert_list")
+    getconf = simple_file("insights_commands/getconf_pagesize")
```

Comment:
Its simple_file should be `insights_commands/getconf_PAGE_SIZE`, you could get its correct format via the `mangle` tool of the core. E.g.

~~~
# source bin/activate
# mangle "/usr/bin/getconf PAGE_SIZE"
insights_commands/getconf_PAGE_SIZE
~~~

Reply:
And you may want to name it as `getconf_page_size`